packages/kbn-grid-layout/grid/grid_layout.tsx

```diff
@@ -152,12 +151,6 @@ export const GridLayout = ({
           rowIndex={rowIndex}
           renderPanelContents={renderPanelContents}
           gridLayoutStateManager={gridLayoutStateManager}
-          setInteractionEvent={(nextInteractionEvent) => {
```

Comment:
I decided to remove this prop as we already have access to `gridLayoutStateManager` inside the handle, so it makes no sense to pass it down.